Bottom: beb0760e0aa35f5ecd42e85ab6c36b86f18d826d
Top:    52d02cb4f737e5c42017e03749150023b8a90ca3
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2021-11-09 11:16:38 +0000

0012-bsp-initial-honister-support

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index 299a0f74bb..358692cd67 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus dunfell gatesgarth"
+LAYERSERIES_COMPAT_{{=machine}} = "honister"
